### PR TITLE
Add some new CSS handles and elements

### PR DESCRIPTION
**Pull request by klzns**
#### What is the purpose of this pull request?

- New element that is invisible by default: Address title.
- CSS Handles: `packageAddressTitle` and `packageDeliveryTitle`
- Variables to `store/header.thanks` message with the user `firstName` and `lastName`.

#### What problem is this solving?

Rebase and fixes https://github.com/vtex-apps/order-placed/pull/142

(About the `displayProducts` prop, one can hide with the CSS Handle `productList` and CSS `display: none;`)

#### How should this be manually tested?

https://breno--storecomponents.myvtex.com/checkout/orderPlaced/?og=1119571095058

#### Screenshots or example usage

image

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by camilavcoutinho — APPROVED** [review]
LGTM on the localization side

File: react/components/DeliveryHeader.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ const CSS_HANDLES = [
   'packageSLA',
   'packageGiftDescription',
   'packageAddressWrapper',
+  'packageAddressTitle',
+  'packageDeliveryTitle',
 ]
```
Comment (victorhmp) [inline react/components/DeliveryHeader.tsx:22]:
```suggestion
] as const
```

File: react/ConfirmationTitle.tsx [hunk 2]
```diff
@@ -2,16 +2,26 @@ import React, { FC } from 'react'
 import { FormattedMessage } from 'react-intl'
 import { useCssHandles } from 'vtex.css-handles'
 
+import { useOrderGroup } from './components/OrderGroupContext'
+
 const CSS_HANDLES = ['confirmationTitle']
```
Comment (victorhmp) [inline react/ConfirmationTitle.tsx:7]:
```suggestion
const CSS_HANDLES = ['confirmationTitle'] as const
```

**Review by victorhmp — APPROVED** [review]

**Comment by klzns** [comment]
@allcontributors add @razvanudream for code

**Pull request merged**